testing/sdk_auth.py

```diff
@@ -204,33 +160,44 @@ def _copy_file_to_localhost(self):
         raise RuntimeError("Failed to chown keytab file: {}".format(repr(e)))
 
     # 3. Copy from private agent to leader
-    scp_cmd = "scp {linux_user}@{kdc_host_name}:/home/{linux_user}/{keytab_file} /home/{linux_user}/{keytab_file}".format(
+    src = "{linux_user}@{kdc_host_name}:/home/{linux_user}/{keytab_file}".format(
         linux_user=LINUX_USER,
         kdc_host_name=self.kdc_host_name,
         keytab_file=self.keytab_file_name
     )
-    cmd = "eval $(ssh-agent -s) && ssh-add /ssh/key && dcos node ssh --master-proxy --leader --option StrictHostKeyChecking=no '{}'".format(scp_cmd)
+    dest = "/home/{linux_user}/{keytab_file}".format(
+        linux_user=LINUX_USER,
+        keytab_file=self.keytab_file_name
+    )
+    scp_cmd = "scp -o StrictHostKeyChecking=no {src} {dest}".format(src=src, dest=dest)
+    ssh_context = "eval $(ssh-agent -s) && ssh-add /ssh/key"
```

Comment:
When I tried this locally, this failed in the first call to `run_command_on_agent` due to my SSH key not having been added. We could run this earlier in this method.

Reply:
Don't need SSH anymore as we just fetch the file. Removed all of this.